docsrv/fetcher_test.go

```diff
@@ -0,0 +1,28 @@
+package docsrv
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	testOwner   = "erizocosmico"
```

Comment:
I realized that here your're not testing git, but testing the api, so it make sense using a real repo.